platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/text/LocalGlyphRasterizer.java

```diff
@@ -0,0 +1,24 @@
+package com.mapbox.mapboxsdk.text;
+
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Bitmap;
+import android.graphics.Typeface;
+
+public class LocalGlyphRasterizer {
+
+    public static Bitmap drawGlyphBitmap(String fontFamily, boolean bold, char glyphID) {
```

Comment:
Annotated with `@WorkerThread` to indicate it won't be called on the main thread?

Reply:
Android always uses a dedicated render thread now, right? Because this is called from the render thread, not from the tile worker threads...

Reply:
Yes, we only support a-synchronous rendering on Android now.